### PR TITLE
Upgrade coral to resolve catalog correctly for underlying tables

### LINKED ISSUE #5785: Cannot access Hive view when Hive catalog name is not "hive"
Starting from version 345 can't get hive views from Presto when catalog name not `hive`: 
`Caused by: java.sql.SQLException: Query failed (#20201102_203650_09667_pba5b): line 1:15: Failed analyzing stored view 'myhive.test.view_hive': line 2:6: Catalog 'hive' does not exist`

It works in version 344

### PR DESCRIPTION
**Pull request by phd3**
Fix #5785 

This PR is to gather early feedback on the testing. The coral version needs to be upgraded after https://github.com/linkedin/coral/pull/19 is merged and released.

### CODE REVIEW

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 1]
```diff
@@ -101,10 +103,15 @@ public void testShowCreateView()
...
 
-        // view SQL depends on Hive distribution
-        assertThat(query("SHOW CREATE VIEW hive_show_view")).hasRowsCount(1);
+        QueryResult result = query("SHOW CREATE VIEW hive_show_view");
+
+        assertThat(result).hasRowsCount(1);
+        String actualSql = (String) result.row(0).get(0);
+        assertEquals(
+                actualSql.replaceAll("\n", " ").replaceAll("\\s+", " "),
+                "CREATE VIEW hive.default.hive_show_view AS SELECT \"n_nationkey\" FROM \"default\".\"nation\"");
```
Comment (kokosing) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:114]: we need to use something else that `hive` catalog name here.

Comment (findepi) [reply presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:114]: In product tests we have few hive catalogs mapped to same metastore.
So you can access the Hive view from two of them within single test.

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 2]
```diff
@@ -101,10 +103,15 @@ public void testShowCreateView()
     {
         onHive().executeQuery("DROP VIEW IF EXISTS hive_show_view");
 
-        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT * FROM nation");
```
Comment (findepi) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:104]: what was wrong about `*`?

can we test both cases, with `*` and with explicit column(s)?

(nation table has few columns, it doesn't hurt to enumerate all of them)

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 3]
```diff
@@ -101,10 +103,15 @@ public void testShowCreateView()
...
+        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT n_nationkey FROM nation");
 
-        // view SQL depends on Hive distribution
-        assertThat(query("SHOW CREATE VIEW hive_show_view")).hasRowsCount(1);
+        QueryResult result = query("SHOW CREATE VIEW hive_show_view");
+
+        assertThat(result).hasRowsCount(1);
+        String actualSql = (String) result.row(0).get(0);
+        assertEquals(
+                actualSql.replaceAll("\n", " ").replaceAll("\\s+", " "),
```
Comment (findepi) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:113]: keeping `actualSql` intact is OK.

also, if we used to output nicely formatted result and in the future we stop doing that, i would like the test to tell me so.

```suggestion
                actualSql,
```

**Comment by alexjo2144** [comment]
Coral release `1.0.12` went up last night with the fix

**Comment by phd3** [comment]
@alexjo2144 yes, I'll post an update soon.

**Comment by alexjo2144** [comment]
Thanks, realized as soon as I commented that you're also at LinkedIn and probably already knew that...

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 4]
```diff
@@ -100,11 +103,26 @@ public void testExistingView()
...
+                ", \"n_comment\"\n" +
+                "FROM\n" +
+                "  \"default\".\"nation\"";
+
+        QueryResult actualResult = query(format(showCreateViewSql, "hive"));
+        assertThat(actualResult).hasRowsCount(1);
+        assertEquals((String) actualResult.row(0).get(0), format(expectedResult, "hive"));
+
+        // Validate the translated view sql for a catalog name other than "hive"
+        actualResult = query(format(showCreateViewSql, "hive_with_external_writes"));
```
Comment (alexjo2144) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:123]: Might be worth calling out in the comment that the `hive_with_external_writes` points to the same HMS instance

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**